.dockerignore

```diff
@@ -0,0 +1,26 @@
+## MAC files ##
```

Comment:
Note, copied over from git ignore file (don't think it matters that there's file/path names that don't exist in my repo)